subsys/bluetooth/services/fast_pair/fp_crypto/Kconfig.fp_crypto

```diff
@@ -42,6 +42,24 @@ config BT_FAST_PAIR_CRYPTO_OBERON
 	help
 	  Select Oberon cryptographic backend for Fast Pair.
 
+config BT_FAST_PAIR_CRYPTO_NRF_SECURITY
```

Comment:
I wonder if it would not be nicer and more straightforward to name it `CRYPTO_PSA` like it has been done in the Matter: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-2.6.0.html#matter (`CONFIG_CHIP_CRYPTO_PSA`)

Reply:
Good idea. Changed.